# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/thunderbird/hotfix-rustc189.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

see also: https://bugzilla.mozilla.org/show_bug.cgi?id=1982003

--- thunderbird-142.0/third_party/rust/allocator-api2/src/stable/vec/mod.rs.vanilla	2025-08-26 22:29:13.354883179 +0200
+++ thunderbird-142.0/third_party/rust/allocator-api2/src/stable/vec/mod.rs	2025-08-26 22:29:21.230481216 +0200
@@ -51,6 +51,8 @@
 //!
 //! [`push`]: Vec::push
 
+#![warn(dangerous_implicit_autorefs)]
+
 #[cfg(not(no_global_oom_handling))]
 use core::cmp;
 use core::cmp::Ordering;
